tinylog-api-kotlin/src/main/kotlin/org/tinylog/kotlin/TaggedLogger.kt

```diff
@@ -790,11 +790,33 @@ class TaggedLogger internal constructor(private val tag: String?) {
 	 * Functions that produce the arguments for formatted text message
 	 */
 	fun error(exception: Throwable, message: String, vararg arguments: () -> Any?) {
-		if (minimumLevelCoversError) {
-			provider.log(stackTraceDepth, tag, Level.ERROR, exception, formatter, message, *arguments.asSuppliers())
+		errorTags.forEach {
+			provider.log(stackTraceDepth, it, Level.ERROR, exception, formatter, message, *arguments.asSuppliers())
 		}
 	}
 
+	/**
+	 * Checks if any of the tags in a given {@link Set} is enabled for a given {@link Level}.
+	 *
+	 * @param tags  {@link Set} of tags to check
+	 * @param level the log level that at least one of the tags must be enabled
+	 * @return {@code true} if the level is enabled for at least one of the tags in the given {@link Set}. otherwise, {@code false}
+	 */
+	private fun anyEnabled(tags: Set<String?>, level: Level): Boolean {
+		return tags.any{provider.isEnabled(stackTraceDepth, it, level)}
```

Comment:
The method `anyEnabled()` adds a stack trace element to the stack trace, so `stackTraceDepth + 1` is required.